features/src/autogluon/features/binning.py

```diff
@@ -57,7 +57,7 @@ def generate_bins(X_features: DataFrame, features_to_bin: list, ideal_bins: int
                 is_satisfied = True
                 # print('max_iterations met, stopping prior to satisfaction!', column, len(interval_index))
 
-        bins_final = interval_index.right.values
+        bins_final = np.array(interval_index.right.values, dtype=np.float64)
```

Comment:
Avoid using float64, it uses too much memory. Keep as whatever it was before (int?)

Reply:
This is still float64. What was it before?